recipes/peewee/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "2.8.1" %}
+
+package:
+  name: peewee
+  version: {{ version }}
+
+source:
+  fn: peewee-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/peewee/peewee-{{ version }}.tar.gz
+  md5: 542f5c3e7792c14da6c5bc385f90ca77
+
+build:
+
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - gcc
```

Comment:
You need to add `isl 0.12.*` here and at run time otherwise `gcc` won't work. See https://github.com/conda-forge/isl-feedstock/issues/3